src/DataModelRegistry.hpp

```diff
@@ -26,9 +26,9 @@ class NODE_EDITOR_PUBLIC DataModelRegistry
 
   struct TypeConverterItem
   {
-    RegistryItemPtr Model;
-    NodeDataType    SourceType;
-    NodeDataType    DestinationType;
+    RegistryItemPtr Model{};
+    NodeDataType    SourceType{};
+    NodeDataType    DestinationType{};
```

Comment:
But yes, if we would add some `int` to the `NodeDataType` struct we would become some garbage without a proper initialization.